pennylane_ionq/device.py

```diff
@@ -109,16 +121,25 @@ def reset(self):
         self._prob_array = None
         self.histogram = None
         self.circuit = {
+            "format": "ionq.circuit.v0",
             "qubits": self.num_wires,
             "circuit": [],
             "gateset": self.gateset,
         }
         self.job = {
-            "lang": "json",
-            "body": self.circuit,
+            "input": self.circuit,
             "target": self.target,
             "shots": self.shots,
         }
+        if self.error_mitigation is not None:
+            self.job["error_mitigation"] = self.error_mitigation

```

Comment:
Codecov is upset that this line isn't being tested. updating a test to pass a value to this kwarg should be enough.

Reply:
coverage added